src/config/index.ts

```diff
@@ -12,12 +13,12 @@ const config = {
     'https://ipfs.everipedia.org/ipfs/',
   infuraId: process.env.NEXT_PUBLIC_INFURA_ID,
   alchemyApiKey: String(process.env.NEXT_PUBLIC_ALCHEMY_API_KEY),
-  alchemyChain: process.env.NEXT_PUBLIC_ALCHEMY_CHAIN || 'maticmum',
+  alchemyChain: process.env.NEXT_PUBLIC_ALCHEMY_CHAIN_POLYGON || 'maticmum',
   graphqlUrl:
     process.env.NEXT_PUBLIC_EP_API || 'https://api.dev.braindao.org/graphql',
   chainId: process.env.NEXT_PUBLIC_CHAIN_ID || '80001',
   chainName: process.env.NEXT_PUBLIC_CHAIN_NAME || 'Mumbai',
-  ensRPC: String(process.env.NEXT_PUBLIC_ENS_RPC),
+  ensRPC: String(process.env.NEXT_PUBLIC_ENS_RPC_POLYGON),
```

Comment:
NEXT_PUBLIC_ENS_RPC_POLYGON that endpoint is ethereum mainnet, name is wrong. There is not ENS on polygon

Reply:
@kesar I don't really understand this.

Reply:
name of constant should be NEXT_PUBLIC_ENS_RPC not NEXT_PUBLIC_ENS_RPC_POLYGON because the endpoint is connecting to Ethereum mainnet, not to Polygon chain